docs/getting-started-guides/scratch.md

```diff
@@ -514,38 +515,18 @@ To run an etcd instance:
 1. make any modifications needed
 1. start the pod by putting it into the kubelet manifest directory
 
-### Apiserver
+### Apiserver, Controller Manager, and Scheduler
 
-To run the apiserver:
-1. select the correct flags for your cluster
-1. write a pod spec for the apiserver using the provided template
-1. start the pod by putting it into the kubelet manifest directory
-
-Here are some apiserver flags you may need to set:
-  - `--cloud-provider=`
-  - `--cloud-config=` if cloud provider requires a config file (GCE, AWS). If so, need to put config file into apiserver image or mount through hostPath.
-  - `--address=${MASTER_IP}`.  
-   - or `--bind-address=127.0.0.1` and `--address=127.0.0.1` if you want to run a proxy on the master node.
-  - `--cluster-name=$CLUSTER_NAME`
-  - `--service-cluster-ip-range=$SERVICE_CLUSTER_IP_RANGE`
-  - `--etcd-servers=http://127.0.0.1:4001`
-  - `--tls-cert-file=/srv/kubernetes/server.cert` -%}
-  - `--tls-private-key-file=/srv/kubernetes/server.key` -%}
-  - `--admission-control=$RECOMMENDED_LIST`
-    - See [admission controllers](../admin/admission-controllers.md) for recommended arguments.
-  - `--allow-privileged=true`, only if you trust your cluster user to run pods as root.
- 
-If you are following the firewall-only security approach, then use these arguments:
-  - `--token-auth-file=/dev/null`
-  - `--insecure-bind-address=$MASTER_IP`
-  - `--advertise-address=$MASTER_IP`
+The apiserver, controller manager, and scheduler will each run as a pod on the master node.
 
-If you are using the HTTPS approach, then set:
-  - `--client-ca-file=/srv/kubernetes/ca.crt`
-  - `--token-auth-file=/srv/kubernetes/known_tokens.csv`
-  - `--basic-auth-file=/srv/kubernetes/basic_auth.csv`
+For each of these components, the steps are similar:
 
-*TODO* document proxy-ssh setup.
+1. start with a provided template for a pod 
+1. set the `APISERVER_IMAGE`, `CNTRLMNGR_IMAGE`, and `SCHEDULER_IMAGE to the values chosen in [Selecting Images](#selecting-images).
+1. determine which flags are needed for your cluster, using the advice below each template.
+1. set the `APISERVER_FLAGS`, `CNTRLMNGR_FLAGS, and `SCHEDULER_FLAGS` to the space-separated list of flags for that component.
+1. start the pod by putting the completed template into the kubelet manifest directory
```

Comment:
Ah, I see that maybe this was intentional. I guess it's OK, since the person has to read through the whole doc anyway.